larray/io/array.py

```diff
@@ -196,6 +196,7 @@ def read_csv(filepath_or_buffer, nb_index=None, index_col=None, sep=',', headers
          FO  2  3
     """
     if not np.isnan(na):
+        fill_value = na
         warnings.warn("Argument na will be replace by fill_value in a future version of larray.",
```

Comment:
I would rather use the "deprecated" or "renamed" wording, like we did for deprecated functions. Here it is only an argument but the concept is the same. Something like: 

    read_csv `na` argument has been renamed to `fill_value`. Please use that instead.